version.html

```diff
@@ -0,0 +1 @@
+Latest release
```

Comment:
I think you can leave this off, I'd find it a bit distracting and not clear that it's referring to the version of the page itself. It's more important that the latest development version over in https://github.com/dafny-lang/dafny/pull/2700 is clearly labelled.

Reply:
Done